From 81b051224c65bd6943bb912e62f4a6c29a905c19 Mon Sep 17 00:00:00 2001
From: Zongchun Yu <zongchun.yu@nxp.com>
Date: Fri, 19 Jun 2020 07:23:14 +0530
Subject: [PATCH] ls1012a-frwy: Update kernel address in input_uimage_secure

ls1012afrwy has wrong kernel address entry in 'input_uimage_secure'.
Update 'input_uimage_secure' with correct kernel address as per kernel
address in u-boot for ls1012afrwy.

Signed-off-by: Zongchun Yu <zongchun.yu@nxp.com>
---
 .../uni_sign/ls104x_1012/ls1012afrwy/input_uimage_secure      | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/input_files/uni_sign/ls104x_1012/ls1012afrwy/input_uimage_secure b/input_files/uni_sign/ls104x_1012/ls1012afrwy/input_uimage_secure
index 3bd1bfc..a95c1f0 100644
--- a/input_files/uni_sign/ls104x_1012/ls1012afrwy/input_uimage_secure
+++ b/input_files/uni_sign/ls104x_1012/ls1012afrwy/input_uimage_secure
@@ -10,7 +10,7 @@ ESBC=1
 ---------------------------------------------------
 # Entry Point/Image start address field in the header.[Mandatory]
 # (default=ADDRESS of first file specified in images)
-ENTRY_POINT=81000000
+ENTRY_POINT=92000000
 ---------------------------------------------------
 # Specify the file name of the keys seperated by comma.
 # The number of files and key select should lie between 1 and 4 for 1040 and C290.
@@ -36,7 +36,7 @@ IMAGE_TARGET=
 ---------------------------------------------------
 # Specify IMAGE, Max 8 images are possible. DST_ADDR is required only for Non-PBL Platform. [Mandatory]
 # USAGE : IMAGE_NO = {IMAGE_NAME, SRC_ADDR, DST_ADDR}
-IMAGE_1={uImage.bin,81000000,ffffffff}
+IMAGE_1={uImage.bin,92000000,ffffffff}
 IMAGE_2={,,}
 IMAGE_3={,,}
 IMAGE_4={,,}
-- 
2.17.1

